integration_tests/dbt_project.yml

```diff
@@ -145,6 +163,81 @@ seeds:
         system_modstamp: timestamp
         is_deleted: boolean
 
+    sf_account_history_data:
+      +column_types:
+        annual_revenue: float
+        last_activity_date: timestamp
+        last_referenced_date: timestamp
+        last_viewed_date: timestamp
+        annual_revenue: float
+        is_deleted: boolean
+    sf_contact_history_data:
+      +column_types:
+        birthdate: timestamp 
+        last_activity_date: timestamp 
+        last_modified_date: timestamp 
+        last_viewed_date: timestamp
+        is_deleted: boolean
+    sf_event_history_data:
```

Comment:
Same comment about removing the history configs for models we are not bringing forward anymore.

Reply:
Removed